aea/cli/fetch.py

```diff
@@ -46,8 +46,6 @@
 def fetch(click_context, public_id, alias, local):
     """Fetch Agent from Registry."""
     if local:
-        ctx = cast(Context, click_context.obj)
```

Comment:
Sure this can be removed?

Reply:
I've moved this to _fetch_agent_locally, the only place it's used.